bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/CallOptionsFactory.java

```diff
@@ -58,7 +59,12 @@
     @Override
     public <RequestT> CallOptions create(MethodDescriptor<RequestT, ?> descriptor,
         RequestT request) {
-      return CallOptions.DEFAULT;
+      Deadline contextDeadline = Context.current().getDeadline();
```

Comment:
Should we make Default CallOptionsFactory just return CallOptions.DEFAULT, like what the comment says?

Reply:
I'll update the comments.  We should honor the context deadline.

Reply:
I've updated the comments.